assets/js/types/type-defs/stock-status.ts

```diff
@@ -0,0 +1,5 @@
+export type StockStatus = 'instock' | 'outofstock' | 'onbackorder';
+
+export type stockStatusOptions = {
```

Comment:
I guess that it is a typo

```suggestion
export type StockStatusOptions = {
```

Reply:
It is. Thanks for spotting it!